code/game/machinery/_machinery.dm

```diff
@@ -170,7 +170,7 @@ Class Procs:
 /obj/machinery/Destroy()
 	GLOB.machines.Remove(src)
 	end_processing()
-	dump_contents()
+	dump_inventory_contents()
```

Comment:
Is this still required? It feels like Destroy() should just annihilate objects and clean up references and not bother "creating" anything as this seems to do.

Reply:
I decided to play it on the safe side and not have qdeling a machine qdel its inventory contents. I can qdel everything in it including player mobs and contents all the same, then it would be up to external code to make sure it's not deleting things it shouldn't be when qdelling a machine.